### ISSUE TITLE
How to launch the application with a french interface?

### ISSUE THREAD

**Issue by devEric69** [tomboy-notes/tomboy-ng#238]
Hello,

I've installed Tomboy-ng on Ubuntu 20.04LTS (french native). The application runs actually with an english interface. So, I'm trying to switch the interface towards the french translation. To do this, I've modified its launcher with "tomboy-ng --lang=fr" (adding the parameter. --lang=fr, to force the language; I saw that there is a tomboy-ng.fr.po file in french). Nevertheless, the interface remains in english. So, what should\could I do in order to translate it into French, please?

I remain available for any tests.
Regards.

**Comment by davidbannon** [comment]
And that sure is an interesting one. Firstly, just having your local default set to French should be enough. And certainly adding --lang=fr should override any local defaults. It does for me, here.
Now, the French translation is new and somewhat incomplete. So, lets be exact here.
1. In the opening splash screen, three buttons across the top Menu, Hide and Quit are not translated. But the text below the buttons is in French, see image -
![Fr](https://user-images.githubusercontent.com/6291248/115553855-39794900-a2f1-11eb-97ed-4f171a46f496.png)
    Do you see any French in your case ?
2. tomboy-ng comes with a set of help notes, we do not, as yet, have a French version.
3. Maybe a useful test, can you please try Spanish ?  --lang=es

The person who very kindly did the initial French translation was seriously distracted by Covid, I have not been in touch with him or her since then, do hope things are OK.  But more work is certainly needed there.

Davo

**Comment by davidbannon** [comment]
Oh, another thing I should have asked.  Which version of tomboy-ng are you using ?  French has not been there for very long.   And, perhaps important, where did you get it from ?  Here on Github, the Ubuntu PPA, Debian ?
Davo

**Comment by devEric69** [comment]
Hello @Davo,

>
>  Which version of tomboy-ng are you using ?
>

here is the explanation of the solution: I had on my computer a version 0.30 which is convenient for me, with an english interface. For a person who is uncomfortable with english, I've tried the french translation... Except that, as you have pointed out and according to the changelogs, the french came later! So, I've reinstalled tomboy-ng version 0.32 and "voila": the interface is in french. Yippee!

Sorry for this useless question.

![tomboy_french_OK](https://user-images.githubusercontent.com/5680812/115676654-18941080-a350-11eb-95e8-60bf921a5f0d.png)

Regards.
